src/main/java/seedu/address/model/task/Task.java

```diff
@@ -92,10 +92,27 @@ public void addPerson(Person person) {
         people.add(person);
     }
 
+    /**
+     * Remove a person from the list of people associated with the task.
+     *
+     * @param person Person to remove
+     */
     public void removePerson(Person person) {
         people.remove(person);
     }
 
+    /**
+     * Update a person in the list of people associated with the task.
+     *
+     * @param person Person to add
+     */
+    public void updatePerson(Person person, Person editedPerson) {
```

Comment:
missing out one more @param editedPerson

Reply:
👌  thanks for reviewing!